pkg/operator/sync.go

```diff
@@ -283,12 +284,22 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		if err != nil {
 			klog.Warningf("could not find configmap specified in image.config.openshift.io/cluster with the name %s", cfg.Spec.AdditionalTrustedCA.Name)
 		} else {
-			for key, d := range cm.Data {
-				raw, err := base64.StdEncoding.DecodeString(d)
+			newKeys := make([]string, len(cm.Data))
```

Comment:
suggestion: A shortcut is to use the `sets` package in `k8s.io/apimachinery/pkg/util/sets`. This could then become something like:

```golang
newKeys := sets.StringKeySet(cm.Data).List()
```

We already have this package vendored into the MCO so it would take no additional work to use it.